lighthouse.md

```diff
@@ -0,0 +1,66 @@
+# Project Lighthouse
+
+Project Lighthouse is conceptually linked to Project Carmel. They may appear as separate entities, but both projects can potentially share similar technical foundations. While Carmel is focused on debates, Lighthouse takes those principles and applies them to the management of a knowledge base. The underlying architecture can be similar, promoting synergies and allowing both projects to benefit from shared enhancements and updates.
+
+Draft spec, please provide feedback in the form of pull requests and issues to github.com/allinbits/carmel.
+
+## Initial terms
+
+ * All contributors agree on a curator or a panel of curators.
```

Comment:
Can the curators list be updated from time to time or is a permanent position? Also the contributors can agree on removing any of them in case it's required for whichever the reason is (absence, lack of opinion, not efficient enough, biased on opinions, etc)?

Reply:
Inspired by Jae's linked statements, we could create a Git-like setup where non-curators can fork and rebase.

I'm open to changing curators, provided the original team gets a fair chance to finish their task.